datadog-iam-user-handler/src/datadog_iam_user/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "2.0.0"
```

Comment:
maybe 1.2.0 instead ?
```suggestion
__version__ = "1.2.0"
```
last released version was 1.1.0 https://github.com/DataDog/datadog-cloudformation-resources/releases/tag/datadog-iam-user-1.1.0

Reply:
I think we should keep the bump to 2.0.0 since we're also updating the schema to remove the State models and changing number -> int

Reply:
Maybe 2.0.0-beta.1 though

Reply:
Yeah, since there are no changes to the schema, I think we can do a minor. Though for now I'll just bump this to dev anyway and we can bump to the proper version at release time.

Reply:
```suggestion
__version__ = "1.2.0.dev"
```